modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -219,6 +219,19 @@ supports up to `64` threads. If this field is empty, the Ingress Controller uses
 
 * `tunnelTimeout` specifies how long a tunnel connection, including websockets, remains open while the tunnel is idle. If unset, the default timeout is `1h`.
 
+* `maxConnections` specifies the maximum number of simultaneous connections that can be established per HAProxy process. Increasing this value lets each ingress controller pod handle more connections at the cost of additional system resources. Permitted values are `0`, `-1`, any value within the range `2000` and `2000000`, or the field can be left empty. 
+
+** If this field is left empty or has the value `0`, the ingress controller will use the default value of `20000`. 
```

Comment:
We should note that the default of 20,000 is subject to change in future releases.

Reply:
thanks, added